src/Azure.Functions.Cli/Kubernetes/KEDA/V2/KedaV2Resource.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure.Functions.Cli.Kubernetes.KEDA.V2.Models;
+using Azure.Functions.Cli.Kubernetes.Models;
+using Azure.Functions.Cli.Kubernetes.Models.Kubernetes;
+using Newtonsoft.Json.Linq;
+
+namespace Azure.Functions.Cli.Kubernetes.KEDA.V2
+{
+    public class KedaV2Resource
+    {
+        public IKubernetesResource GetKubernetesResource(string name, string @namespace, TriggersPayload triggers,
```

Comment:
Is there a reason why you are using `GetKubernetesResource` and not more concrete what it will generate?

KEDA has multiple Kubernetes resources (`ScaledJob`, `ScaledObject`, `TriggerAuthentication`) which is why I preferred a more explicit name